src/Sarif.WorkItemFiling/WorkItemFields.cs

```diff
@@ -24,5 +24,10 @@ public static class WorkItemFields
         /// Title field name.
         /// </summary>
         public const string Title = "System.Title";
+
+        /// <summary>
+        /// Repro Steps field name.
+        /// </summary>
+        public const string ReproSteps = "Microsoft.VSTS.TCM.ReproSteps";
```

Comment:
This seems specific to ADO, perhaps like AzureDeveOpsFilingTarget this class should be AzureDevOpsWorkItemFields as it seems specific to ADO and not generic across different filing targets.